evm-adapters/src/sputnik/cheatcodes/cheatcode_handler.rs

```diff
@@ -113,11 +113,17 @@ pub(crate) fn convert_log(log: Log) -> Option<String> {
         LogNamedBytes32Filter(inner) => {
             format!("{}: 0x{}", inner.key, hex::encode(inner.val))
         }
-        LogNamedDecimalIntFilter(inner) => format!(
-            "{}: {:?}",
-            inner.key,
-            ethers::utils::parse_units(inner.val, inner.decimals.as_u32()).unwrap()
-        ),
+        LogNamedDecimalIntFilter(inner) => {
```

Comment:
Can you add a test for this functionality [here](https://github.com/gakonst/foundry/blob/7f0c3ade4bc0003101a5466e6d7fe2c2c51e20c0/evm-adapters/src/sputnik/cheatcodes/cheatcode_handler.rs#L1715) and by adding a negative log emit in `DebugLogs.sol`?

Reply:
Sure thing